app/models/user.rb

```diff
@@ -380,6 +380,10 @@ def organization_and_child_ids
     taxonomy_and_child_ids(:organizations)
   end
 
+  def my_taxonomies_ids
```

Comment:
Are this method and its test necessary now? They look like leftovers from the previous implementation